src/components/SelectionList/Search/ReportListItem.tsx

```diff
@@ -109,6 +109,13 @@ function ReportListItem<TItem extends ListItem>({
         return null;
     }
 
+    const reportItemTotal = reportItem?.total ?? 0;
+    const reportItemUnheldTotal = reportItem?.unheldTotal ?? 0;
+    const isOnHold = reportItemTotal - reportItemUnheldTotal === 0;
+
+    const isApproveAction = reportItem.action === CONST.SEARCH.ACTION_TYPES.APPROVE;
+    const shouldUseSuccessStyle = isApproveAction ? !isOnHold : true;
+
```

Comment:
Can we use `ReportUtils.hasHeldExpenses` like so?
```suggestion
    const hasHeldExpenses = ReportUtils.hasHeldExpenses('', reportItem.transactions);
```